tests/test_cli.py

```diff
@@ -7,7 +7,8 @@
 import re
 
 profiles_dir = os.path.join(Path.cwd(), "tests/mock/mockProfile")
-project_dir = os.path.join(Path.cwd(), "tests/mock")
+# project_dir = os.path.join(Path.cwd(), "tests/mock")
+project_dir = "/Users/gorkemyurtseven/dev/duck_db_project/duckdb_dbt_project"
```

Comment:
🤨

Reply:
i was just running the tests sorry not ready for review!